<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientQT</class>
 <widget class="QMainWindow" name="clientQT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clientQT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonConnect">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="statutConnectLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bouttonSendMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="celciusButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="farenheitButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>°F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hygrometrieButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>hygrométrie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditASK">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditANSWER">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boutonConnect</sender>
   <signal>clicked()</signal>
   <receiver>clientQT</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouttonSendMessage</sender>
   <signal>clicked()</signal>
   <receiver>clientQT</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>celciusButton</sender>
   <signal>clicked()</signal>
   <receiver>clientQT</receiver>
   <slot>onCelciusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>farenheitButton</sender>
   <signal>clicked()</signal>
   <receiver>clientQT</receiver>
   <slot>onFarenheitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hygrometrieButton</sender>
   <signal>clicked()</signal>
   <receiver>clientQT</receiver>
   <slot>onHygrometrieClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onCelciusClicked()</slot>
  <slot>onFarenheitClicked()</slot>
  <slot>onHygrometrieClicked()</slot>
 </slots>
</ui>
